photon-core/src/main/java/org/photonvision/vision/pipe/impl/Calibrate3dPipe.java

```diff
@@ -113,9 +124,128 @@ protected CameraCalibrationCoefficients process(
             return null;
         }
 
+        for (int i = 0; i < rvecs.size(); i++) {
+            // logger.info("OpenCV found board at R="+rvecs.get(i).dump()+"t="+tvecs.get(i).dump());
+        }
+
         JsonMatOfDouble cameraMatrixMat = JsonMatOfDouble.fromMat(cameraMatrix);
         JsonMatOfDouble distortionCoefficientsMat = JsonMatOfDouble.fromMat(distortionCoefficients);
 
+        var observations =
+                createObservations(in, cameraMatrix, distortionCoefficients, rvecs, tvecs, null);
+
+        // Standard deviation of results
+        try {
+            // Print calibration successful
+            logger.info(
```

Comment:
Log is done for OpenCV but not MrCal, this will lead to confusion when reviewing logs. Method should be made to print a generic message based on Calib result or create another log statement for calib w/MrCal